### ISSUE TITLE
Inquiry about missing file epropnp_det.apis in EPro-PnP-Det

### ISSUE THREAD

**Issue by a1445605672** [tjiiv-cprg/EPro-PnP#31]
During execute process, it was found that init_detector, inference_detector, show_result file were missing
![图片](https://user-images.githubusercontent.com/41555798/186300936-35a9c82d-8a9a-4125-9e7a-d23f18c21797.png)
![01](https://user-images.githubusercontent.com/41555798/186301173-436f24c2-905a-4c86-a80f-762310ff9e29.jpg)
![图片](https://user-images.githubusercontent.com/41555798/186301152-de43f41a-f975-4628-9721-abbd4853136a.png)

**Comment by Lakonik** [comment]
Hi! Can you give the full error message and some information of the environment (e.g. PyTorch version, etc.)?

**Comment by a1445605672** [comment]
Is this because there are no init_detector, inference_detector, show_result files in the epropnp_det.apis directory?
![图片](https://user-images.githubusercontent.com/41555798/186320490-ae8cd5ba-88e1-4889-8ece-576f68bbbc6e.png)
![图片](https://user-images.githubusercontent.com/41555798/186320796-6eb0f6b4-b74c-491a-96be-56f0b64e6afe.png)

**Comment by Lakonik** [comment]
No. This seems to be a CUDA version mismatch between MMCV, PyTorch and your current nvidia driver.
